docs/extensions/extension_sandboxing.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Extension Sandboxing
+category: 62c5cb137dbdad00536291a6
+hidden: false
+---
+
+Phylum's CLI extensions allow developers to impose additional restrictions when
+running third party applications. This can protect the system from damage when
+these applications contain vulnerabilities or execute untrusted code.
+
+## Example
+
+The following code provides an example on how you could sandbox `cat` to only
+allow access to files in the current working directory or below it:
```

Comment:
```suggestion
allow access to files in the current working directory or its descendants:
```
(optional)

Reply:
I think "descendants" might be more difficult to understand for some users than "below it". Especially because it can be interpreted as some kind of derived directory rather than a hierarchical structure.

I'm open to other suggestions, but I don't think descendants is the way to go.